src/Ros2ForUnity/Scripts/PostInstall.cs

```diff
@@ -47,6 +51,18 @@ public void OnPostprocessBuild(BuildReport report)
             FileUtil.CopyFileOrDirectory(
                 r2csMeta, outputDir + "/" + execFilename + "_Data/Plugins/x86_64/" + r2csMetadataName);
         }
+
+        // Copy versioned libraries on Linux (Unity skips them)
+        if (ROS2ForUnity.GetOS() == ROS2ForUnity.Platform.Linux) {
```

Comment:
I think that you could put that code inside `if` in line [43](https://github.com/RobotecAI/ros2-for-unity/blob/develop/src/Ros2ForUnity/Scripts/PostInstall.cs#L43)

Reply:
Done